backend/app.py

```diff
@@ -1,12 +1,19 @@
 # flask_web/app.py
 
 from flask import Flask
-app = Flask(__name__)
+from flask_sqlalchemy import SQLAlchemy
+from pathlib import Path
+
+SQLITE_FILE = '/db/digitalesWartenDB.sqlite'
 
-@app.route('/')
-def hello_world():
-    return 'Hey, we have Flask in a Docker container!'
+app = Flask(__name__)
 
+app.config['SQLALCHEMY_DATABASE_URI'] = f'sqlite:///{SQLITE_FILE}'
+db = SQLAlchemy(app)
 
-if __name__ == '__main__':
-    app.run(debug=True, host='0.0.0.0')
+if not Path.is_file(SQLITE_FILE):
+    Path(SQLITE_FILE).parent.absolute().mkdir(parents=True, exist_ok=True)
```

Comment:
```suggestion
db_file = Path(SQLITE_FILE)
if not db_file.is_file():
    db_file.parent.absolute().mkdir(parents=True, exist_ok=True)
```
Or just leave out the

```suggestion
Path(SQLITE_FILE).parent.absolute().mkdir(parents=True, exist_ok=True)
```
mkdir should be file anyways

Reply:
I just tested it, when running `Path('a/b.foo').mkdir(parents=True)` a folder named `b.foo` is created